apiClientDotNet/Utils/RestRequestHandler.cs

```diff
@@ -15,94 +12,79 @@ namespace apiClientDotNet.Utils
 {
     public class RestRequestHandler
     {
-        public X509CertificateCollection Certificates
-        {
-            get;
-            protected set;
-        }
+        public X509CertificateCollection Certificates { get; protected set; }
 
         public void AddCert(string fileName, string password)
```

Comment:
I don't understand why do we need this method, because the Certificates variable is initialized at line 61. I guess it needs refactoring